exec/java-exec/src/main/resources/rest/profile/profile.ftl

```diff
@@ -132,7 +132,7 @@
                 <#list model.getOptionList() as option>
                   <tr>
                     <td>${option.getName()}</td>
-                    <td>${option.getValue()?c}</td>
+                    <td>${option.getValue()?string}</td>
```

Comment:
I am not familiar with FreeMarker so I do not understand the change. In Drill-4792 you mentioned 
> Since org.apache.drill.exec.server.options.OptionValue.getValue() returns Object, Freemarker built-in c is used to convert Object to string.
Could you please explain why that was not sufficient and how using `string` changes that?